app/[locale]/institute/historical-footprint/page.tsx

```diff
@@ -0,0 +1,28 @@
+import Heading from '~/components/heading';
+import { getTranslations } from '~/i18n/translations';
+
+export default async function HistoricalFootprint({
+  params: { locale },
+}: {
+  params: { locale: string };
+}) {
+  const text = (await getTranslations(locale)).Institute;
+  return (
+    <>
```

Comment:
Uh we don't need this if there's only a since child in `return`. But instead of removing this, remove `<main>` and apply `container` on both the heading and `p`